Change what is accepted by ModelRc::new and ModelRc::from (#904)
sixtyfpsui/sixtyfps: api/sixtyfps-rs/migration.md

}
```

#### ModelHandle

`ModelHandle` was renamed [`ModelRc`].

> [`ModelRc::new`]  no longer takes a `Rc`, but takes the structure that implements model directly.
> To construct a `ModelRc` from a Rc, one should use the `From` trait. So [`ModelRc::from`] is doing what
> `ModelHandle::new` was doing.

Reviewer: ```suggestion
[`ModelRc::new`]  no longer takes a `Rc`, but takes the structure that implements the [`Model`] trait directly.
To construct a `ModelRc` from a Rc for your model, use the `From` trait. [`ModelRc::from`] is doing what
`ModelHandle::new` was doing.
```
